code/modules/virus2/effect/stage_4.dm

```diff
@@ -166,15 +166,16 @@
 		var/mob/living/carbon/human/H = mob
 		for (var/datum/organ/external/E in H.organs)
 			if (E.status & ORGAN_BROKEN && prob(30))
-				E.status ^= ORGAN_BROKEN
+				to_chat(user, "<span class = 'notice'>You feel the bones in your [E.display_name] set back into place.</span>")
```

Comment:
H, not user

Reply:
good spot, I'm sleepycoding so out of habit put user.